ci-operator/step-registry/openshift/e2e/openstack/customnoupgrade/openshift-e2e-openstack-customnoupgrade-workflow.yaml

```diff
@@ -0,0 +1,17 @@
+workflow:
+  as: openshift-e2e-openstack-customnoupgrade
+  steps:
+    pre:
+    - ref: ipi-conf-customnoupgrade
```

Comment:
What about having `ipi-conf-customnoupgrade` always in the `ipi-openstack-pre` chain, and having it turn into a no-op when both variables are `""`?

Reply:
That does feel more 'composed'.

Reply:
That said, I think I recall getting pushback when I tried doing something similar before which resulted in me creating the ugly `-noramfs` ref. Lets see who objects.